pyproject.toml

```diff
@@ -72,8 +73,12 @@ typing = [
     "typing_extensions>=4.0.0"
 ]
 all = [
-    "astropy[recommended]",  # installs the [recommended] dependencies
+    # Install grouped optional dependencies
+    "astropy[recommended]",
+    "astropy[docs]",
```

Comment:
`test_all` has already been discussed, but I don't think `all` dependencies should contain `docs` dependencies either.